spec/arrays.dd

```diff
@@ -884,10 +884,12 @@ printf("the string is '%.*s'\n", str.length, str.ptr);
         $(P The best way is to use std.stdio.writefln, which can handle
         D strings:)
 
+$(RUNNABLE_EXAMPLE
 ---------
 import std.stdio;
 writefln("the string is '%s'", str);
 ---------
+)
```

Comment:
This one doesn't work. (For one, a statement outside a function, for two, referencing a variable declared in another block.)

I thought you used a script to check which example works and enable it only for those? Or did you choose examples manually?